packages/office-ui-fabric-react/src/components/Keytip/Keytip.styles.ts

```diff
@@ -0,0 +1,77 @@
+import { IKeytipStyleProps, IKeytipStyles } from './Keytip.types';
+import { ICalloutContentStyleProps, ICalloutContentStyles } from '../../Callout';
+import { IStyleFunction, IPoint } from '../../Utilities';
+
+export const getStyles = (props: IKeytipStyleProps): IKeytipStyles => {
+  const { theme, disabled, visible } = props;
+  return {
+    container: [
+      {
+        backgroundColor: theme.palette.neutralDark
+      },
+      disabled && {
+        opacity: 0.5,
```

Comment:
Do we have an specific HC styling? Disabled Keytips should be setting the color: GrayText and opacity: 1